src/controllers/shared.controllers/director.nationality.controller.ts

```diff
@@ -23,21 +23,26 @@ export const getDirectorNationality = async (req: Request, res: Response, next:
     const session: Session = req.session as Session;
     const officerFiling = await getOfficerFiling(session, transactionId, submissionId);
     let currentUrl: string;
+    let directorName: string;
+
     if (isUpdate) {
```

Comment:
Perhaps the logic for getting a name based on CH01/AP01 could be a reusable function? As it's used in a few controllers.